presentation/src/main/java/com/sopt/presentation/calendar/component/CalendarGroupItem.kt

```diff
@@ -0,0 +1,77 @@
+package com.sopt.presentation.calendar.component
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.sopt.core.designsystem.theme.NoostakAndroidTheme
+import com.sopt.core.designsystem.theme.NoostakTheme
+import com.sopt.core.extension.noRippleClickable
+import com.sopt.domain.entity.CalendarGroupEntity
+import com.sopt.presentation.group.component.GroupImage
+
+@Composable
+fun CalendarGroupItem(
+    modifier: Modifier = Modifier,
+    data: CalendarGroupEntity,
+    isSelected: Boolean,
+    onClick: () -> Unit = {}
+) {
+    Column(
+        modifier = modifier
+            .width(56.dp)
+            .noRippleClickable { onClick() }
+            .then(
+                if (isSelected) {
+                    Modifier.graphicsLayer {
+                        alpha = 1f
+                    }
+                } else {
+                    Modifier.graphicsLayer { alpha = 0.6f }
+                }
```

Comment:
P2: 요렇게 한 줄로 줄이면 더 간결하고 가독성이 좋아질 것 같습니다!
```suggestion
.graphicsLayer(alpha = if (isSelected) 1f else 0.6f)
```